### PR TITLE
Add quantized  JIT example

### PR DESCRIPTION
**Pull request by calmdown13**
## Summary
Small adaptation of the JIT example to demonstrate how to load quantized models.

### CODE REVIEW

File: examples/jit-quantized/main.rs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    // Load the image file and resize it to the usual imagenet dimension of 224x224.
+    let image = imagenet::load_image_and_resize224(image_file)?;
+
+    // Load the Python saved module.
+    let model = tch::CModule::load(model_file)?;
+
+    // Time inference
+    let now = SystemTime::now();
+    for _ in 1..10 {
+        image.unsqueeze(0).apply(&model);
```
Comment (LaurentMazare) [inline examples/jit-quantized/main.rs:35]: Could you fix the warning there about the output being unused, e.g. adding a `let _output =` at the beginning of this line?

File: examples/jit-quantized/resnet.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import torch
+import torchvision
+
+
+def save_resnet18_script_module(
+    filename: str, pretrained: True, quantize: False
```
Comment (LaurentMazare) [inline examples/jit-quantized/resnet.py:6]: I'm not very familiar with python type annotations, but do True/False make sense in this setting or should these be `bool` (or default values, e.g. `pretrained=True`.

File: examples/jit-quantized/main.rs (hunk 1 above)
Comment (calmdown13) [reply examples/jit-quantized/main.rs]: sure, updated

File: examples/jit-quantized/resnet.py (hunk 2 above)
Comment (calmdown13) [reply examples/jit-quantized/resnet.py]: woops my bad, updated

**Review by LaurentMazare — APPROVED** [review]

**Comment by LaurentMazare** [comment]
Merged, thanks for the PR!

**Pull request merged**